components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/src/EIAnalyticsHorizontalBarChart.jsx

```diff
@@ -25,7 +25,6 @@ const BAR_GRAPH_TYPE = 'Component Type Selection';
 const URL_PARAMETER_ID = 'id';
 const DIV_ID_GRAPH = 'graph';
 const PUBLISHER_DATE_TIME_PICKER = 'granularity';
-const TENANT_ID = '-1234';
```

Comment:
Does the default value available from props? What if meta tenant id is not configured?

Reply:
In this PR, I am also adding -1234 as tenantId in the EIAnalytics dashboard JSON as default value. (commit bd46cd4).
 If a user wants to create another EI Dashboard with a different tenantId, user has to edit the properties.tenantId in the json and create a new dashboard. That's how @lasanthaS told me when we had an offline discussion yesterday.

Reply:
Noted